command/sub8_alarm/alarm_handlers/kill.py

```diff
@@ -8,13 +9,23 @@ class Kill(HandlerBase):
     def __init__(self):
         self._killed = False
         self._last_mission_killed = False
+        self.bagging_server = rospy.ServiceProxy('/online_bagger/dump', BaggerCommands)
 
     def raised(self, alarm):
         self._killed = True
+        self.bagger_dump()
 
     def cleared(self, alarm):
         self._killed = False
 
+    def bagger_dump(self):
+        ''' Dump online_bagger'''
```

Comment:
I believe PEP8 calls for docstrings in double quotes (which I did not do) and:

 PEP8:
"For one liner docstrings, please keep the closing """ on the same line. "